packages/cubejs-backend-native/src/node_export.rs

```diff
@@ -425,6 +631,60 @@ fn exec_sql(mut cx: FunctionContext) -> JsResult<JsValue> {
     Ok(promise.upcast::<JsValue>())
 }
 
+fn sql4sql(mut cx: FunctionContext) -> JsResult<JsValue> {
```

Comment:
Don't you mind moving all these functions to sub module (like it was done for orchestrator/planner/python)? - To keep this file clean and hold only kind of entry points, register interfaces, while having all the handlers in a separate sub modules.